lib/debug/actor-inspector/inspector-panel.js

```diff
@@ -247,22 +247,22 @@ target.on("onRegisterPanels", registrar => {
 });
 
 target.on("onUnregisterPanels", registrar => {
-  updateRegistration(registrar);
+  registrar.unregisterPanel(InspectorPanel);
```

Comment:
@Honza I've re-checked this line (using the steps in the pull request description) and the ```updateRegistration``` function enters the wrong branch during the addon shutdown and it try to register the RDP panel instead of unregistering it (because ```dev``` and ```theme``` are both ```true```).

the simpler solution I've found is ```registrar.unregisterPanel```, which ensures that we're going to remove the panel if it exists, and it will became a noop if the panel is not active.